ddtrace/llmobs/_integrations/openai.py

```diff
@@ -185,15 +185,15 @@ def _llmobs_set_meta_tags_from_chat(
             output_messages = []
             for idx, choice in enumerate(resp.choices):
                 content = getattr(choice.message, "content", "")
-                if content:
-                    output_messages.append({"content": str(content), "role": choice.message.role})
-                elif getattr(choice.message, "function_call", None):
+                if getattr(choice.message, "function_call", None):
                     content = choice.message.function_call.arguments
-                    output_messages.append({"content": str(content), "role": choice.message.role})
                 elif getattr(choice.message, "tool_calls", None):
+                    content = ""
                     for tool_call in choice.message.tool_calls:
-                        content = tool_call.function.arguments
-                        output_messages.append({"content": str(content), "role": choice.message.role})
+                        content += "\n{}\n\n{}\n".format(
```

Comment:
If were adding extra newlines is there a .strip() we should do at the end when the content is formed?

Also since were doing custom formatting for the user here, is it worth thinking about adding something like
[tool: <tool_name>] or something? Similar to how we do datashield placeholders? Just to give the user some signal we're doing custom formatting for them. Don't have any exact ideas on how it would apply here but just a thought

Reply:
Good call, will follow consistency for how we format image data!